### ISSUE TITLE
installation on Python 2.7

### ISSUE THREAD

**Issue by ROZBEH** [machinalis/iepy#102]
Hi all,

I don't know whether this is a general problem or just occurred on my MAC with python 2.7. As I was trying to run the installation of iepy from setup.py I received an error about encoding issue. It had some problem with the encoding. After some googling it turns out that this is a problem because of Python 2.7. As a result, I did the following change to the setup.py file and it solved my problem. There are two `open()` commands and I changed them to `io.open` and it saved me the time.

Please let me know if I wrong about it.
Actually, this is not necessarily a bug. I just wanted to share it with you and have your idea about it.

Best,
Rouzbeh

**Comment by jmansilla** [comment]
Hi again Rozbeh.

I'm very ashamed of the time I made you waste.

On the mailing list you asked if was it possible to install IEPY under MacOS with python2.7, and I answered yes, but I was wrong. 

I'm sorry, but I was wrong[1] and the truth is that IEPY is only for python3.

You still may try to run IEPY on MacOS, but with python3 installed.


[1] I was thinking on other of the projects we maintain (featureforge) which has support for both python2.7 and python3
